File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/ReaderTagObject.cs
using System;

namespace Library.Reader.Objects
{
    public class ReaderTagObject
    {
        public ReaderTagObject(string hostName, string location, string antennaID, string tagID, DateTime time_occured)
        {
            Host = hostName;
            Location = location;
            AntennaID = antennaID;
            TagID = tagID;
            TimeOccured = time_occured;
        }

        public string Host { get; set; }
        public string Location { get; set; }
        public string AntennaID { get; set; }
        public string TagID { get; set; }
        public DateTime TimeOccured { get; set; }
    }
}

File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/ReaderTagObject.vb
Namespace Objects
    Public Class ReaderTagObject
        Public Sub New(ByVal hostName As String, ByVal location As String, ByVal antennaID As String, ByVal tagID As String, ByVal time_occured As DateTime)

            Me.Host = hostName
            Me.Location = location
            Me.AntennaID = antennaID
            Me.TagID = tagID
            Me.TimeOccured = time_occured

        End Sub

        Public Property Host As String

        Public Property Location As String

        Public Property AntennaID As String

        Public Property TagID As String

        Public Property TimeOccured As DateTime

        'Public Property Message As String
    End Class
End Namespace
